### PR TITLE
[KRK-2023] Add fixed event data

### PR DESCRIPTION
**Pull request by MilyMilo**
Hi again,

I had to iron out some details mentioned in the previous PR #12197 before submitting. We also had to change the location in the meanwhile, sorry for the delay!

Hopefully this is all right now!

### CODE REVIEW

File: content/events/2023-krakow/welcome.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    </p>
+    <p style="font-weight: 800">Why is it worth to join us?</p>
+    <p>
+      Our guests come from various technical backgrounds, including software developers, senior software engineers, CTOs/CEOs, testers, IT Admins and IT Managers. This diverse group of experts will share their knowledge and experiences on leading internet projects with the latest trends in management. Come and join us to learn and share the latest IT solutions and cultural experiences in the tech community.
+    </p>
+  </div>
+  <div class="text">
+    <a href="https://eventory.cc/event/devopsdays-krakow-2023" type="button" class="btn btn-danger btn-lg btn-block">REGISTRATION</a>
+    <a href="https://dodkrakow.pl/call-for-papers-2023/" type="button" class="btn btn-danger btn-lg btn-block">PROPOSE</a>
+    <a href="mailto:olga.michaliszyn@proidea.pl " type="button" class="btn btn-danger btn-lg btn-block">SPONSOR</a>
```
Comment (yvovandoorn) [inline content/events/2023-krakow/welcome.md:54]: I don't see this person being listed as an organizer. Either have this route to a listed organizer or the mailing list.

File: data/events/2023-krakow.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+location: "Qubus Hotel Kraków"
+location_address: "Nadwiślańska 6, 30-527 Kraków"
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: registration
+    url: "https://eventory.cc/event/devopsdays-krakow-2023"
+  - name: propose
+    url: "https://dodkrakow.pl/call-for-papers-2023/"
+  - name: sponsor
+    url: "mailto:olga.michaliszyn@proidea.pl"
```
Comment (yvovandoorn) [inline data/events/2023-krakow.yml:42]: I don't see this person being listed as an organizer. Either have this route to a listed organizer or the mailing list.

**Comment by yvovandoorn** [comment]
Hi there @MilyMilo!

A few things. 
1. There is a mismatch of organizers currently on the mailing list (`krakow@devopsdays.org`) versus in the PR. Please submit a complete list of organizers (and the confirmation of organizer removals) to `info@devopsdays.org`. 
2. Someone who is not listed as an organizer is listed as the contact for sponsors. Either have that person join as an organizer, change the email to an organizer or change it to the city mailing list. 
3. Per _People will usually need at least 4-6 weeks to arrange for travel or time off, and you’ll want your call to be open for at least a month, and you’ll want at least 2 weeks to consider proposals and fill in any gaps. This means that you should open your CFP as soon as possible, and close it at least 6-8 weeks before your event._ located @ https://devopsdays.org/organizing#call-for-proposals, this event is scheduled in two months. As this is no longer an online event, it would be good to understand if you've already received proposals. 

It might be helpful to reach out to myself and @KrisBuytaert on the devopsdays Slack on the last point. But please address points 1 and 2 for the merge to be accepted.

**Comment by MilyMilo** [comment]
Hi there, thanks for the response!

As per 1 & 2 - I've just emailed the full team roster so we can sort out slack and mailing list access. I'll include Olga on the team page.  

In regards to point 3: Yes we do already have proposals lined-up - we're waiting on the event to be listed on the dod page to fully start it, so we can start receiving the proposals via the proper channel. The timeline is a bit tight due to some changes in venue but we're confident it's going to work.

I'm fixing the issues addressed, let me know if you need anything else - we'd really like to go forward to launch the CFP and start selling tickets!

**Comment by ivellios** [comment]
@yvovandoorn, @phrawzty - I believe this PR addresses all your concerns. We also requested adding Olga to the mailing list. 
Can you please take another look at the PR and review the code? From my perspective, the information in this PR is accurate and current.

**Review by ivellios — APPROVED** [review]
All information looks good from my pov

**Comment by phrawzty** [comment]
Merged as per discussions in email and Slack.

**Pull request merged**